packages/core/content-releases/server/src/controllers/release-action.ts

```diff
@@ -17,6 +20,62 @@ const releaseActionController = {
       data: releaseAction,
     };
   },
+  async findMany(ctx: Koa.Context) {
+    const releaseId: GetReleaseActions.Request['params']['releaseId'] = ctx.params.releaseId;
+
+    const allowedContentTypes = getAllowedContentTypes({ strapi, userAbility: ctx.state.userAbility });
+    
+    const morphSanitizedPopulate = await allowedContentTypes.reduce(async (promiseAccumulator, contentTypeUid) => {
+      const accumulator = await promiseAccumulator;
+
+      const permissionsManager = await strapi.admin.services.permission.createPermissionsManager({
+        ability: ctx.state.userAbility,
+        model: contentTypeUid
+      });
+
+      // We create this to filter populated entries based on custom conditions on permissions
+      accumulator[contentTypeUid] = {
+        filters: permissionsManager.getQuery(CONTENT_MANAGER_READ_ACTION)
+      };
+
+      return accumulator;
+    }, Promise.resolve({} as Record<UID.ContentType, { filters: object }>));
+
+
+    const releaseService = getService('release', { strapi });
+    const { results, pagination } = await releaseService.findActions(releaseId, allowedContentTypes, { 
+      populate: {
+        entry: {
+          on: morphSanitizedPopulate
+        }
+      }
```

Comment:
what populate object will the admin need to send?

Reply:
Maybe missing some context!
But I believe the populate object might not be necessary here, it was originally used in the content manager to populate the creator fields automatically, and also sanitize unauthorized populates. But it doesn't seem you are receiving any populate object here to be filtered out.

Reply:
I see , from what you pointed out in Slack, this is made to manage custom conditions :)